src/pages/Home/index.tsx

```diff
@@ -1,12 +1,14 @@
-import * as React from 'react'
+import React from 'react'
+import {Link} from 'react-router-dom'
 import {useTranslation} from 'react-i18next'
 
 const Home = () => {
 	const [t] = useTranslation()
 
 	return (
 		<div data-testid="home-page">
-			<h2>{t('common.welcome')}</h2>
+			<h2>{t('appName')}</h2>
+			<Link to="/logout">Logout</Link>
```

Comment:
Hmm..I don't think Logout deserve to be a separate component. I think we can have a click handler here and call `logout` action when user click to the link